server.js

```diff
@@ -21,9 +15,116 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = listEndpoints(app)
+  res.json(endpoints);
+});
+
+// Shark attacks endpoint to receive a collection of all shark attacks
+app.get("/shark-attacks", (req, res) => {
+  res.json(globalSharkAttackData)
 });
 
+// Shark attacks "id" endpoint, (used the original_order in json), to receive a single shark attack
+app.get("/shark-attacks/:id", (req, res) => {
+  const {id} = req.params
+  const sharkAttack = globalSharkAttackData.find(sharkAttack => sharkAttack.original_order === +id)
+
+  if (sharkAttack) {
+    res.json(sharkAttack)
+  } else {
+  res.status(404).send("Unfortunately for you, but luckily for the swimmers - No shark attack was found!")
```

Comment:
Since you're displaying the error message here using the send method instead of the json method, it looks a little different from the other error messages. No biggy, just wanted to mention it 🥰